node_macros/src/rust_type.rs

```diff
@@ -39,27 +49,45 @@ impl RustType {
             .clone()
     }
 
-    /// Returns the type without the path, but with type arguments, e.g. `SmallNet<NodeId>`.
-    pub fn ty(&self) -> TokenStream {
-        let ident = self.ident();
-        let args = &self
+    /// Returns the types identifier with type arguments attached as strings, i.e.
+    /// `GossipIncomingDeploy`'.
```

Comment:
Not sure if I understand correctly:

```suggestion
    /// Returns the type's identifier with type arguments concatenated into a single camel case
    /// identifier, e.g. `GossipIncomingDeploy`.
```